src/objects.js

```diff
@@ -147,3 +147,61 @@ export class Page {
     return new Page(this.data.map(func), this.before, this.after)
   }
 }
+
+/** FaunaDB time. See the [docs](https://faunadb.com/documentation/queries#values-special_types). */
+export class FaunaTime {
+  /**
+   * @param {string|Date} value If a Date, this is converted to a string.
+   */
+  constructor(value) {
+    if (value instanceof Date)
+      value = value.toISOString()
```

Comment:
The native Date (actually a time) type doesn't have nanosecond precision, so we can't use it for FaunaTime.
We could use Date for FaunaDate, although we only want date precision while js's Date goes up to millis.
So since JS only has the one type, and it's in between FaunaDate and FaunaTime in precision, it seemed better to not use it directly.